src/TesApi.Web/Controllers/TaskServiceApi.cs

```diff
@@ -95,7 +95,8 @@ public virtual async Task<IActionResult> CancelTask([FromRoute][Required] string
                     }
                     catch (RepositoryCollisionException exc)
                     {
-                        // TODO
+                        logger.LogError(exc, $"RepositoryCollisionException in CancelTask for {id}");
```

Comment:
A similar logging entry should be made in the similar catch statement body in `src/TesApi.Web/Scheduler.cs`. That location doesn't require any additional failure returns because the system will try again automatically.